packages/calling/calling/src/CallingClient/registration/register.ts

```diff
@@ -69,7 +70,8 @@ export class Registration implements IRegistration {
   private rehomingIntervalMax: number;
   private mutex: Mutex;
   private metricManager: IMetricManager;
-  private emitter: CallingClientEmitterCallback;
+  private emitter: LineEmitterCallback;
```

Comment:
call this lineEmitter now that we calliClient one as well

Reply:
Done